Remove`learn/configuration/settings.md` (#2044)
meilisearch/documentation: learn/core_concepts/documents.md

If a field contains an object, Meilisearch flattens it during indexing using dot notation and brings the object's keys and values to the root level of the document itself. This flattened object is only an intermediary representation—you will get the original structure upon search. You can read more about this in our [dedicated guide](/learn/advanced/datatypes.md#objects).

> With [ranking rules](/learn/core_concepts/relevancy.md#ranking-rules), you can decide what fields are more relevant than others. For example, you may decide recent movies should be more relevant than older ones. You can also configure how Meilisearch handles certain fields at an [index level](/reference/api/settings.md) in the settings.

Reviewer: Did not think the link was adding anything here—distracting, if anything—so replaced it with a segue to the next section.

Also, not the right PR to fix this... but this is a bad example! Making recent movies more relevant than older ones is an example of sorting on a single field. "Deciding which fields are more relevant than others," in the first sentence, alludes to attribute ranking order.

I may ask you to review this guide next week @maryamsulemani97 .

```suggestion
With [ranking rules](/learn/core_concepts/relevancy.md#ranking-rules), you can decide which fields are more relevant than others. For example, you may decide recent movies should be more relevant than older ones. You can also designate certain fields as displayed or searchable.
```
